### PR TITLE
Add Lists sync macOS settings

### PR DESCRIPTION
**Pull request by kuvaibhavmsft**
Adding documentation for Nucleus admin controls on macOS.

### CODE REVIEW

File: SharePoint/SharePointOnline/lists-sync-policies-macos.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+search.appverid:
+- SPO160
+- MET150
+ms.collection:  
+- M365-collaboration
+description: "Learn how to control Lists sync by using app preferences on macOS."
+---
+# Use macOS preferences to control Lists sync settings
+
+This article describes macOS preferences for Microsoft Lists (and SharePoint lists). For info about controlling OneDrive sync settings on macOS, see [Deploy and configure OneDrive on macOS](/sharepoint/deploy-and-configure-on-macos).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/lists-sync-policies-macos.md:23]:
```suggestion
This article describes macOS preferences for Microsoft Lists (and SharePoint Lists). For info about controlling OneDrive sync settings on macOS, see [Deploy and configure OneDrive on macOS](/sharepoint/deploy-and-configure-on-macos).
```

File: SharePoint/SharePointOnline/lists-sync-policies-macos.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+## Manage Lists sync settings on macOS using property list files
+
+Lists sync gets packaged, installed, and updated through the OneDrive sync standalone (non-App Store) app's existing update mechanism. Due to this, some Lists sync preferences are listed under OneDrive group domain. After the OneDrive sync app for Mac is installed, users can configure settings for Lists sync. As an administrator, you might want to provide users in your organization with a standard set of preferences. Preferences for the Lists sync app for Mac are stored in property list (.plist) files.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/lists-sync-policies-macos.md:27]:
```suggestion
Lists sync gets packaged, installed, and updated through the OneDrive sync standalone (non-App Store) app's existing update mechanism. Due to this, some Lists sync preferences are listed under the OneDrive group domain. After the OneDrive sync app for Mac is installed, users can configure settings for Lists sync. As an administrator, you might want to provide users in your organization with a standard set of preferences. Preferences for the Lists sync app for Mac are stored in property list (.plist) files.
```

File: SharePoint/SharePointOnline/lists-sync-policies-macos.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+|:-----|:-----|:-----|
+|**.plist location**  |~/Library/Preferences/com.microsoft.SharePoint-mac.plist  |~/Library/Group Containers/UBF8T346G9.OneDriveStandaloneSuite/Library/Preferences/UBF8T346G9.OneDriveStandaloneSuite.plist  |
+
+## List of policies by string ID
+
+- (DisableNucleusSync) [Prevent Lists sync from running on the device](lists-sync-policies-macos.md#prevent-lists-sync-from-running-on-the-device)
+
+- (BlockExternalListSync) [Prevent users from syncing lists shared from other organizations](lists-sync-policies-macos.md#prevent-users-from-syncing-lists-shared-from-other-organizations)
+
+- (DisableNucleusSilentConfig) [Prevent users from getting silently signed in to Lists sync with existing Microsoft account credentials being used across Microsoft apps on macOS](lists-sync-policies-macos.md#prevent-users-from-getting-silently-signed-in-to-lists-sync-with-existing-microsoft-account-credentials-being-used-across-microsoft-apps-on-macos)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/lists-sync-policies-macos.md:39]:
```suggestion
- (DisableNucleusSync) [Prevent Lists sync from running on the device](lists-sync-policies-macos#prevent-lists-sync-from-running-on-the-device)

- (BlockExternalListSync) [Prevent users from syncing lists shared from other organizations](lists-sync-policies-macos#prevent-users-from-syncing-lists-shared-from-other-organizations)

- (DisableNucleusSilentConfig) [Prevent users from getting silently signed in to Lists sync with existing Microsoft account credentials being used across Microsoft apps on macOS](lists-sync-policies-macos#prevent-users-from-getting-silently-signed-in-to-lists-sync-with-existing-microsoft-account-credentials-being-used-across-microsoft-apps-on-macos)
```

File: SharePoint/SharePointOnline/lists-sync-policies-macos.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+### Prevent Lists sync from running on the device
+
+By default, Lists sync is turned on for users of Microsoft Lists. If you set this preference, Lists sync will be blocked from running on the device.
+
+> Location: [Lists Sync preferences](lists-sync-policies-macos.md#manage-lists-sync-settings-on-macos-using-property-list-files)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/lists-sync-policies-macos.md:45]:
```suggestion
**Location**: [Lists Sync preferences](lists-sync-policies-macos#manage-lists-sync-settings-on-macos-using-property-list-files)
```

File: SharePoint/SharePointOnline/lists-sync-policies-macos.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+```xml
+<key>DisableNucleusSync</key>
+<integer>0</integer>
+```
+
+### Prevent users from syncing lists shared from other organizations
+
+Enabling this setting prevents users at your organization from syncing lists that are shared from other organizations. After the setting is enabled (value 1) on a computer, lists shared from other organizations won't sync. Disable the setting (value 0) to allow your users to sync external lists.
+
+> Location: [Lists Sync preferences](lists-sync-policies-macos.md#manage-lists-sync-settings-on-macos-using-property-list-files)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/lists-sync-policies-macos.md:65]:
```suggestion
**Location**: [Lists Sync preferences](lists-sync-policies-macos#manage-lists-sync-settings-on-macos-using-property-list-files)
```

File: SharePoint/SharePointOnline/lists-sync-policies-macos.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+```xml
+<key>BlockExternalListSync</key>
+<integer>0</integer>
+```
+
+### Prevent users from getting silently signed in to Lists sync with existing Microsoft account credentials being used across Microsoft apps on macOS
+
+Lists sync is set up to automatically sign users in with credentials being used across other Microsoft apps on macOS like OneDrive. If you enable this setting, automatic sign-in and Lists sync setup would not occur.
+
+> Location: [OneDrive group preferences](lists-sync-policies-macos.md#manage-lists-sync-settings-on-macos-using-property-list-files)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/lists-sync-policies-macos.md:85]:
```suggestion
**Location**: [OneDrive group preferences](lists-sync-policies-macos#manage-lists-sync-settings-on-macos-using-property-list-files)
```

**Comment by kuvaibhavmsft** [comment]
> Copy review - @kuvaibhavmsft, kindly commit the suggested changes. Many thanks.

Thanks for the feedback @JohanFreelancer9. Adde suggested changes.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@kuvaibhavmsft 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by kuvaibhavmsft** [comment]
Completed via #3674.

**Pull request closed without merging**